js/everything.js

```diff
@@ -402,6 +410,28 @@ function loadTime() {
     }
 }
 
+// Key to the persisted volume level entry in localStorage
+var LOCAL_STORAGE_VOLUME_KEY = "VOLUME_LEVEL";
+
+// The delay to debounce writes of the volume level to localStorage
+var VOLUME_PERSIST_DELAY = 200;
+
+// The timeout ID tracking debounced writes of volume level to localStorage
+var debouncedPersistVolumeTimer;
+
+function storeVolume(volume) {
+    if (debouncedPersistVolumeTimer) {
+        clearTimeout(debouncedPersistVolumeTimer);
+    }
+    debouncedPersistVolumeTimer = setTimeout(function () {
+        localStorage.setItem(LOCAL_STORAGE_VOLUME_KEY, volume);
+    }, VOLUME_PERSIST_DELAY);
+}
+
+function loadVolume() {
+    return parseInt(localStorage.getItem(LOCAL_STORAGE_VOLUME_KEY));
```

Comment:
`parseInt()` should have the second param (radix) set to `10` - the default is 8 and can lead to some oddities. [Good Stack Overflow explanation](http://stackoverflow.com/questions/850341/how-do-i-work-around-javascripts-parseint-octal-behavior)